sonic_data_client/mixed_db_client.go

```diff
@@ -153,14 +170,14 @@ func (c *MixedDbClient) DbDelTable(table string, key string) error {
 }
 
 func NewMixedDbClient(paths []*gnmipb.Path, prefix *gnmipb.Path, zmqAddress string) (Client, error) {
-	var client MixedDbClient
 	var err error
 
 	// Testing program may ask to use redis local tcp connection
 	if UseRedisLocalTcpPort {
 		useRedisTcpClient()
 	}
 
+	var client = getMexedDbClient(zmqAddress)
```

Comment:
We still need to access AppDB non-dash tables. Does this change impact the old feature?

Reply:
@ganglyu , could you please confirm? As I know GNMI is only for Dash.

Reply:
We need to support dash tables and non-dash tables.

Reply:
Fixed
CUrrently the C++ class inheritance information missing in go wrapper, so create a new PR to update sonic-swss-common SWIG file in this PR: https://github.com/sonic-net/sonic-swss-common/pull/794